ang/test/civicase/case/details/directives/case-details.directive.spec.js

```diff
@@ -48,14 +53,45 @@
     });
 
     describe('focusToggle()', function () {
-      beforeEach(function () {
-        compileDirective();
-        element.isolateScope().isFocused = true;
-        element.isolateScope().focusToggle();
+      describe('basic test', function () {
+        beforeEach(function () {
+          compileDirective();
+          element.isolateScope().isFocused = true;
+          element.isolateScope().focusToggle();
+        });
+
+        it('toggles the focus state', function () {
+          expect(element.isolateScope().isFocused).toBe(false);
+        });
       });
 
-      it('toggles the focus state', function () {
-        expect(element.isolateScope().isFocused).toBe(false);
+      describe('when case is unfocused and screen width is less than 1690px', function () {
+        beforeEach(function () {
+          spyOn($rootScope, '$broadcast');
+          spyOn($document, 'width').and.returnValue(1600);
+          compileDirective();
+          element.isolateScope().isFocused = true;
+          element.isolateScope().focusToggle();
+        });
+
+        it('fires an event', function () {
+          expect($rootScope.$broadcast)
+            .toHaveBeenCalledWith('civicase::case-details::unfocused');
+        });
+      });
+
+      describe('when case is unfocused and screen width is more than 1690px', function () {
+        beforeEach(function () {
+          spyOn($rootScope, '$broadcast');
+          spyOn($document, 'width').and.returnValue(1700);
+          compileDirective();
+          element.isolateScope().isFocused = true;
+          element.isolateScope().focusToggle();
+        });
+
+        it('fires an event', function () {
```

Comment:
`it('does not fire the case details unfocused event') `

Notice that you used "it fires", when it should not.